client/signup/steps/design-type-with-store/style.scss

```diff
@@ -134,6 +134,12 @@
 	@include breakpoint( ">480px" ) {
 		display: block;
 	}
+
+	img {
+		display: block;
+		margin: 8.33333% 0 3.3333% 7%;
```

Comment:
I feel like we could use more padding in the cards. I found that this worked nicely:

`margin: 10% auto; width: 85%;`

![image](https://user-images.githubusercontent.com/6981253/30114091-c5e5592c-92e4-11e7-9500-8e7a7335930d.png)

cc @cristelrossignol

Reply:
Agreed!